akka-actor/src/main/scala/akka/actor/Props.scala

```diff
@@ -156,10 +112,9 @@ object Props {
  * }}}
  */
 @SerialVersionUID(2L)
-final case class Props(deploy: Deploy, clazz: Class[_], args: immutable.Seq[Any]) {
+final case class Props(deploy: Deploy = Props.defaultDeploy, clazz: Class[_], args: immutable.Seq[Any]) {
```

Comment:
Why is this needed?
In general adding default params is better in the latter (not initial) parameters